mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -1785,6 +1785,45 @@ func TestAccComputeInstanceConfidentialInstanceConfigMain(t *testing.T) {
 	})
 }
 
+<% unless version = 'ga' -%>
+func TestAccComputeInstanceConfidentialHyperDiskBootDisk(t *testing.T) {
+	t.Parallel()
+
+	context_1 := map[string]interface{}{
+		"name":        fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10)),
+		"confidential_compute": true,
+		"kms_key" : acctest.BootstrapKMSKey(t)
+	}
+
+	context_2 := map[string]interface{}{
```

Comment:
Could we clone context_1 and then change `confidential_compute`'s value instead? I think we're pulling the wrong `name` value and doing extra lookups for `kms_key` since we'll make extra API calls as written